### PR TITLE
Helm deploy: live-1 vs live explain

### PR DESCRIPTION
**Pull request by davidread**
Also:
* try to explain Helm a bit better
* .md lint changes

Related to [#3344](https://github.com/ministryofjustice/cloud-platform/issues/3344)

### CODE REVIEW

File: source/documentation/deploying-an-app/app-deploy-helm.html.md.erb [hunk 1]
```diff
@@ -6,31 +6,39 @@ review_in: 3 months
...
+### Requirements
+
 It is assumed you have the following:
 
- - You have [created an environment for your application][env-create]
- - You have installed [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) on your local machine.
- - You have [authenticated][auth-to-cluster] to the cloud-platform-live-1 cluster.
+- You have [created an environment/namespace for your application][env-create]
+- You have installed [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) on your local machine.
+- You have [kubectl connected][auth-to-cluster] to the cluster which your environment is on - either `live` or `live-1`. You will need to have authenticated and possible also switch context, if you have multiple contexts. Check your environment is listed when you do: `kubectl get namespaces`
```
Comment (AntonyBishop) [inline source/documentation/deploying-an-app/app-deploy-helm.html.md.erb:29]: Typo - "possible" to "possibly"

**Review by AntonyBishop — COMMENTED** [review]
Just left comment on typo

File: source/documentation/deploying-an-app/app-deploy-helm.html.md.erb (hunk 1 above)
Comment (davidread) [reply source/documentation/deploying-an-app/app-deploy-helm.html.md.erb]: fixed

Comment (AntonyBishop) [reply source/documentation/deploying-an-app/app-deploy-helm.html.md.erb]: Cheers

**Review by AntonyBishop — APPROVED** [review]

**Pull request merged**